config/webpack/environment.js

```diff
@@ -52,4 +53,16 @@ environment.loaders.delete('nodeModules');
 
 environment.loaders.append('erb', erb);
 
+if (process.env.HONEYBADGER_API_KEY) {
+  environment.plugins.append(
+    'HoneybadgerSourceMap',
+    new HoneybadgerSourceMapPlugin({
+      apiKey: process.env.HONEYBADGER_API_KEY,
+      assetsUrl: `${process.env.APP_PROTOCOL}${process.env.APP_DOMAIN}/packs/js`,
+      silent: false,
+      ignoreErrors: false,
+      revision: process.env.SOURCE_VERSION || 'master'
```

Comment:
I think using a git commit SHA here would be better since that will be easier for folks to link back to Github where as only a few of us have access to Heroku source versions. To get the git commit you want to access `HEROKU_SLUG_COMMIT`

Reply:
Oh thanks! I honestly thought `SOURCE_VERSION` was going to yield me the commit hash
```suggestion
      revision: process.env.HEROKU_SLUG_COMMIT || 'master'
```